React/Base/RCTTouchHandler.m

```diff
@@ -150,6 +150,11 @@ - (void)_updateReactTouchAtIndex:(NSInteger)touchIndex
   reactTouch[@"locationX"] = @(touchViewLocation.x);
   reactTouch[@"locationY"] = @(touchViewLocation.y);
   reactTouch[@"timestamp"] =  @(nativeTouch.timestamp * 1000); // in ms, for JS
+  
+  if([UIDevice currentDevice].systemVersion.floatValue >= 9) {
+    reactTouch[@"force"] = @(nativeTouch.force);
+    reactTouch[@"maxForce"] =  @(nativeTouch.maximumPossibleForce);
```

Comment:
Can you use the same name as the native API? The only reason to deviate would be to follow an existing web specification but I don't think there's any yet :)